tests/test_benchmark/test_benchmark_integration_with_datasets.py

```diff
@@ -0,0 +1,25 @@
+"""test mteb.MTEB's integration with SentenceTransformers"""
+
+from __future__ import annotations
+
+import logging
+from typing import Union
+
+import pytest
+
+import mteb
+from mteb import MTEB
+from mteb.abstasks import AbsTask
+
+from .mock_models import MockNumpyEncoder
+from .task_grid import TASK_TEST_GRID
+
+logging.basicConfig(level=logging.INFO)
+
+
+@pytest.mark.parametrize("task", TASK_TEST_GRID)
+@pytest.mark.parametrize("model", [MockNumpyEncoder()])
+def test_benchmark_sentence_transformer(task: Union[str, AbsTask], model: mteb.Encoder):
+    """Test that a task can be fetched and run"""
+    eval = MTEB(tasks=[task])
```

Comment:
Since this test is under `test_benchmark`, should it test a `Benchmark` object?

Reply:
There is a test for the benchmark object as well, but test_mteb just seems like we are testing the whole package (which is not entirely untrue). Would love a better name